feat: i18n hr-HR

hrvoj3e opened the pull request:
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [x] The commit message follows our guidelines
- [x] It does not introduce a breaking change or has a description of the breaking change

### Description

Add Croatian language to I18n.

hrvoj3e committed:
feat: i18n hr-HR

Croatian language

seonim-ryu commented:
@hrvoj3e Thanks. I reviewed it and could you add a few more?

* Add language to [README](https://github.com/nhn/tui.editor#-features) : Add language information added to the content of _Features> Internationalization (i18n)_.
* Add language to `table-merged-cell` plugin : It should also be added to the `table-merged-cell`. At the end of [this file](https://github.com/nhn/tui.editor/blob/master/plugins/table-merged-cell/src/js/langs.js), you can add the language value to be added.

If you only deal with two things, I'll merge them.

hrvoj3e committed:
docs: add Croatian to Internationalization section

Add language information added to the content of Features> Internationalization (i18n)

hrvoj3e committed:
feat: add Croatian language hr-HR

hrvoj3e commented:
@seonim-ryu I have added the two requested changes. Is this OK?

seonim-ryu reviewed (approved):
@hrvoj3e Perfect! 👍 It is good to keep the commit message well and write it. Thanks for your contributing. I'll merge this PR. 😃
